pkg/baseline/packageIndex.json

```diff
@@ -9,19 +9,28 @@
       ],
       "_AssemblyVersionInPackageVersion": [
         "Used to understand what version of a package to reference.",
-        "Then raise the package version of the reference so that its at least the baseline version."
+        "Then raise the package version of the reference so that its at least the baseline version.",
+        "In cases where the assembly version did not rev but the package did, use the first package version."
       ]
     },
     "System.Private.ServiceModel": {
       "StableVersions": [
         "4.0.0",
         "4.1.0",
+        "4.1.1",
         "4.3.0",
-        "4.3.1"
+        "4.3.1",
+        "4.4.0",
+        "4.4.1",
+        "4.4.2",
+        "4.5.0"
```

Comment:
Is the plan to add `4.5.1` later?

Reply:
No reason not to do it right now, I'll amend the commit.